Framework/Kernel/src/Matrix.cpp

```diff
@@ -1613,7 +1613,7 @@ void fillFromStream(std::istream &is, Kernel::Matrix<T> &in,
 
 // Symbol definitions for common types
 template class MANTID_KERNEL_DLL Matrix<double>;
```

Comment:
Should be change `MANTID_KERNEL_DLL` to `DLLExport` here so that it's clear that this can only every be an export defintion? It's then also consistent with other cases.